examples/siro_sandbox/app_states/app_state_fetch.py

```diff
@@ -8,126 +8,403 @@
 import numpy as np
 from app_states.app_state_abc import AppState
 from camera_helper import CameraHelper
+from controllers.baselines_controller import (
+    FetchBaselinesController,
+    FetchState,
+)
 from controllers.gui_controller import GuiHumanoidController
 from gui_navigation_helper import GuiNavigationHelper
+from gui_pick_helper import GuiPickHelper
+from gui_throw_helper import GuiThrowHelper
 from hablab_utils import get_agent_art_obj_transform, get_grasped_objects_idxs
 
 from habitat.gui.gui_input import GuiInput
 from habitat.gui.text_drawer import TextOnScreenAlignment
+from habitat_sim.physics import MotionType
+
+COLOR_GRASPABLE = mn.Color3(1, 0.75, 0)
+COLOR_GRASP_PREVIEW = mn.Color3(0.5, 1, 0)
+COLOR_FOCUS_OBJECT = mn.Color3(1, 1, 1)
+COLOR_FETCHER_NAV_PATH = mn.Color3(0, 153 / 255, 255 / 255)
+
+RADIUS_GRASPABLE = 0.2
+RADIUS_GRASP_PREVIEW = 0.35
+RADIUS_FOCUS_OBJECT = 0.15
+RADIUS_FETCHER_NAV_PATH = 0.45
+
+RING_PULSE_SIZE = 0.05
+
+disable_spot = False
```

Comment:
This is handy for local testing if you want to test grasping/throwing without activating the Spot fetch behavior.